addons/audio/allegro5/allegro_audio.h

```diff
@@ -355,6 +359,10 @@ ALLEGRO_KCM_AUDIO_FUNC(size_t, al_get_audio_depth_size, (ALLEGRO_AUDIO_DEPTH con
 ALLEGRO_KCM_AUDIO_FUNC(void, al_fill_silence, (void *buf, unsigned int samples,
       ALLEGRO_AUDIO_DEPTH depth, ALLEGRO_CHANNEL_CONF chan_conf));
 
+ALLEGRO_KCM_AUDIO_FUNC(int, al_get_audio_device_count, (void));
```

Comment:
Bikeshedding I know, but should this be `al_get_num_audio_devices` to be consistent with `al_get_num_video_adapters`, `al_get_num_joysticks` and `al_get_num_display_modes` ?
OTOH, `al_get_channel_count` but that's the only one, and there isn't an `al_get_channel` so it's not quite the same.

Reply:
I agree, should've checked for similar functions